string data
float32 lin_vel
float32 ang_vel
---
int8 result